components/skeleton/skeleton-group-mixin.js

```diff
@@ -0,0 +1,22 @@
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+import { SubscriberRegistryController } from '../../controllers/subscriber/subscriberControllers.js';
+
+export const SkeletonGroupMixin = dedupeMixin(superclass => class extends superclass {
+
+	constructor() {
+		super();
+
+		this._readyToDisplayCached = false;
+		this._skeletonSubscribers = new SubscriberRegistryController(this, 'skeleton', {
+			updateSubscribers: this._checkSubscribersReadyToDisplay.bind(this)
+		});
+	}
+
+	_checkSubscribersReadyToDisplay(subscribers) {
+		const readyToDisplay = ![...subscribers.values()].some(subscriber => subscriber.skeleton);
```

Comment:
If there don't end up being any subscribers, I think this will remain in a skeleton state? But maybe that doesn't matter since the group itself doesn't have any skeleton display?

Reply:
Yeah, I _think_ that's okay? `readyToDisplay` only gets set on the subscribers, so the case where there are no subscirbers seems to be a moot point.

We could add a short circuit `if (!subscribers) { return }` to be very clear.

Reply:
@dlockhart looks like the controller will already short circuit if there are no subscribers...so this seems fine as is.

https://github.com/BrightspaceUI/core/blob/c438ce8e4151f4846cbda0082ffb9d50986938a0/controllers/subscriber/subscriberControllers.js#L103-L115

Reply:
Right, so it's not even going to call `updateSubscribers` on the group if there aren't any. Which I think will still keep the group in its initial skeleton state indefinitely?